data/convert.py

```diff
@@ -0,0 +1,17 @@
+from datetime import datetime
+import pytz
+
+
+def convert(date):
```

Comment:
this module name and function name are meaningless.

describe them properly.

Also I am pretty sure you can find a library which does this.  There is probably even a function in `pytz` which does this.